contracts/prize-strategy/multiple-winners/MultipleWinners.sol

```diff
@@ -1,18 +1,29 @@
 // SPDX-License-Identifier: MIT
-
 pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
 
 import "../PeriodicPrizeStrategy.sol";
 
 contract MultipleWinners is PeriodicPrizeStrategy {
 
+  struct MultipleWinnersPrizeSplit {
+      address target;
+      uint16 percentage;
+  }
+
   uint256 internal __numberOfWinners;
+  
+  MultipleWinnersPrizeSplit[2] internal _prizeSplits;
 
   bool public splitExternalErc20Awards;
 
   event SplitExternalErc20AwardsSet(bool splitExternalErc20Awards);
 
   event NumberOfWinnersSet(uint256 numberOfWinners);
+  
+  event PrizeSplitSet(address indexed target, uint16 percentage);
+  
+  event PrizeSplitDistributed(address indexed target, uint16 percentage, uint256 amount);
```

Comment:
Add @notice natspec

Reply:
Shouldn't it be called `PrizeSplitAwarded` since we use the award function to distribute the prize split?
https://github.com/pooltogether/pooltogether-pool-contracts/blob/4449bb2e4216511b7187b1ab420118c30af39eb7/contracts/prize-pool/PrizePool.sol#L512

Reply:
I dont think we need this. The existing Awarded event should be enough?

Reply:
Removed after adding. As we can just track this distribution using the token transfer and correlating it with a target in the _prizeSplits config array.